### PR TITLE
Lesson 8: Circle

### PR DESCRIPTION
**Pull request by itsabhister**

### CODE REVIEW

File: students/itsabhi/Lesson8/circle.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+class Circle:
+    # Circle class
+    _diameter = None  # This cannot be stored as its not an object attribute
+    _area = None  # This cannot be stored as its not an object attribute
+
+    @property
+    def diameter(self):
+        return self.radius*2
+
+    def __init__(self, the_radius):
```
Comment (natasha-aleksandrova) [inline students/itsabhi/Lesson8/circle.py]: init should always be the first method you define

File: students/itsabhi/Lesson8/circle.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import math
+
+
+class Circle:
+    # Circle class
+    _diameter = None  # This cannot be stored as its not an object attribute
+    _area = None  # This cannot be stored as its not an object attribute
```
Comment (natasha-aleksandrova) [inline students/itsabhi/Lesson8/circle.py]: what is the purpose of these?

File: students/itsabhi/Lesson8/circle.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    def area(self):
+        return (self.radius**2)*math.pi
+
+    @classmethod
+    def from_diameter(cls, diam):
+        # Returns a Circle class object
+        return cls(diam/2)
+
+    def __str__(self):
+        return("Circle with radius: "+str(self.radius))
```
Comment (natasha-aleksandrova) [inline students/itsabhi/Lesson8/circle.py]: you don't need parens with return and don't forget proper spacing:
`return "Circle with radius: " + str(self.radius)`

File: students/itsabhi/Lesson8/circle.py (hunk 2 above)
Comment (itsabhister) [reply students/itsabhi/Lesson8/circle.py]: I might have copied these from the class lesson example:
class C:
    _x = None
    @property
    def x(self):
        return self._x

**Pull request merged**